backend/baseclass.pm

```diff
@@ -342,10 +343,18 @@ sub _invoke_video_encoder ($self, $pipe_name, $display_name, @cmd) {
     $pipe->blocking(!!$bmwqemu::vars{VIDEO_ENCODER_BLOCKING_PIPE});
 }
 
+sub _ffmpeg_banner () { `ffmpeg 2>&1` // '' }
```

Comment:
```suggestion
sub _ffmpeg_banner () { qx{ffmpeg 2>&1} // '' }
```

Reply:
Does this change anything? Why is it preferred? Considering most people know backticks from bash it is likely the easier syntax.

Reply:
I find backticks not very eye-friendly, and we use `qx` in the majority of the cases in os-autoinst, and it's easier to grep for.